scss/components/masonry-block/_masonry-block.scss

```diff
@@ -76,20 +70,13 @@
     }
   }
 
-  &.is-wide.has-image-right:not(.has-background),
-  &.is-column-width.has-image-right {
-    .masonry-block__link {
-      color: color(ash);
-    }
-  }
-
   &.has-columns,
   &.is-full-width {
 
     .masonry-block__description {
       color: #FFF;
       @include grid-media($media-sm) {
-        color: color(ash);
+        color: color(text-active);
```

Comment:
OK, wasn't sure how I'd feel about this change, but i think it looks great. Much cleaner!